Gems/Atom/RHI/Vulkan/Code/Source/Platform/Linux/RHI/WSISurface_Linux.cpp

```diff
@@ -17,15 +18,30 @@ namespace AZ
         {
             Instance& instance = Instance::GetInstance();
 
+            xcb_connection_t *  xcb_connection = NULL;
+#if AZ_LINUX_WINDOW_MANAGER_XCB
+            AzFramework::LinuxXCBConnectionManager::Bus::BroadcastResult(xcb_connection, &AzFramework::LinuxXCBConnectionManager::Bus::Events::GetXCBConnection);
+
```

Comment:
Well the xcb_connection_* xcb_connection should be assigned to nullptr.
Also it should be declared inside of the #if AZ_LINUX_WINDOW_MANAGER_XCB block.
Plus I think it should use the Interface directly
```
xcb_connection_t* xcb_connection = nullptr;
if (auto xcbConnectionManager = AzFramework::LinuxXCBConnectionManagerInterface.Get();
    xcbConnectionManger != nullptr
{
    xcb_connection = xcbConnectionManger->GetXCBConnection()
}
```

Reply:
Updated (with fixes)